templates/nextcloud28-entrypoint.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+
+chown -R www-data:www-data /var/www/html
+chmod -R 775 /var/www/html
+
+# Let /entrypoint.sh not run apache
+sed -i 's/exec "$@"//g' /entrypoint.sh
+
+# create file to override config
+cat > /var/www/html/config/liquid.override.config.php << 'DELIM'
+<?php
+$CONFIG = array (
+  'dbhost' =>  getenv('MYSQL_HOST'),
+  'trusted_domains' => [
+                    "localhost",
+                    getenv('NEXTCLOUD_HOST'),
+                    getenv('LIQUID_BASE_IP')
+],
+  'dbname' => getenv('MYSQL_DATABASE'),
+  'dbpassword' => getenv('MYSQL_PASSWORD'),
+);
+DELIM
+
+# Launch canonical entrypoint
+/entrypoint.sh apache2-foreground
+
+unzip /apps-to-install/sociallogin.zip -d /apps-to-install
```

Comment:
this should be moved in the docker build process, we don't want to be unzipping stuff every container start...